### ISSUE TITLE
Continuous integration is failing

### ISSUE THREAD

**Issue by utaal** [buildo/react-autosize-textarea#6]
```
11-Jun-2015 16:10:29 	Executing command bash -c CONTINUOUS_INTEGRATION=1 npm test || true && ./node_modules/eslint/bin/eslint.js src -f compact > eslint-errors
11-Jun-2015 16:10:29 	
11-Jun-2015 16:10:29 	> react-autosize-textarea@0.2.1 test /data
11-Jun-2015 16:10:29 	> ./node_modules/karma/bin/karma start
11-Jun-2015 16:10:29 	
11-Jun-2015 16:10:30 	INFO [karma]: Karma v0.12.36 server started at http://localhost:9876/
11-Jun-2015 16:10:30 	INFO [launcher]: Starting browser Chrome
11-Jun-2015 16:10:32 	WARN [web-server]: 404: /favicon.ico
11-Jun-2015 16:10:32 	INFO [Chrome 43.0.2357 (Linux 0.0.0)]: Connected on socket 2cymhpNLzU436S1Y9TWA with id 56835897
11-Jun-2015 16:10:33 	7 XSELINUXs still allocated at reset
11-Jun-2015 16:10:33 	SCREEN: 0 objects of 256 bytes = 0 total bytes 0 private allocs
11-Jun-2015 16:10:33 	DEVICE: 0 objects of 96 bytes = 0 total bytes 0 private allocs
11-Jun-2015 16:10:33 	CLIENT: 0 objects of 136 bytes = 0 total bytes 0 private allocs
11-Jun-2015 16:10:33 	WINDOW: 0 objects of 32 bytes = 0 total bytes 0 private allocs
11-Jun-2015 16:10:33 	PIXMAP: 2 objects of 16 bytes = 32 total bytes 0 private allocs
11-Jun-2015 16:10:33 	GC: 4 objects of 16 bytes = 64 total bytes 0 private allocs
11-Jun-2015 16:10:33 	CURSOR: 1 objects of 8 bytes = 8 total bytes 0 private allocs
11-Jun-2015 16:10:33 	TOTAL: 7 objects, 104 bytes, 0 allocs
11-Jun-2015 16:10:33 	2 PIXMAPs still allocated at reset
11-Jun-2015 16:10:33 	PIXMAP: 2 objects of 16 bytes = 32 total bytes 0 private allocs
11-Jun-2015 16:10:33 	GC: 4 objects of 16 bytes = 64 total bytes 0 private allocs
11-Jun-2015 16:10:33 	CURSOR: 1 objects of 8 bytes = 8 total bytes 0 private allocs
11-Jun-2015 16:10:33 	TOTAL: 7 objects, 104 bytes, 0 allocs
11-Jun-2015 16:10:33 	4 GCs still allocated at reset
11-Jun-2015 16:10:33 	GC: 4 objects of 16 bytes = 64 total bytes 0 private allocs
11-Jun-2015 16:10:33 	CURSOR: 1 objects of 8 bytes = 8 total bytes 0 private allocs
11-Jun-2015 16:10:33 	TOTAL: 5 objects, 72 bytes, 0 allocs
11-Jun-2015 16:10:33 	1 CURSORs still allocated at reset
11-Jun-2015 16:10:33 	CURSOR: 1 objects of 8 bytes = 8 total bytes 0 private allocs
11-Jun-2015 16:10:33 	TOTAL: 1 objects, 8 bytes, 0 allocs
11-Jun-2015 16:10:33 	1 CURSOR_BITSs still allocated at reset
11-Jun-2015 16:10:33 	TOTAL: 0 objects, 0 bytes, 0 allocs
11-Jun-2015 16:10:33 	npm ERR! Test failed.  See above for more details.
11-Jun-2015 16:10:33 	[dix] Could not init font path element /usr/share/fonts/X11/cyrillic, removing from list!
11-Jun-2015 16:10:33 	[dix] Could not init font path element /usr/share/fonts/X11/100dpi/:unscaled, removing from list!
11-Jun-2015 16:10:33 	[dix] Could not init font path element /usr/share/fonts/X11/75dpi/:unscaled, removing from list!
11-Jun-2015 16:10:33 	[dix] Could not init font path element /usr/share/fonts/X11/Type1, removing from list!
11-Jun-2015 16:10:33 	[dix] Could not init font path element /usr/share/fonts/X11/100dpi, removing from list!
11-Jun-2015 16:10:33 	[dix] Could not init font path element /usr/share/fonts/X11/75dpi, removing from list!
11-Jun-2015 16:10:34 	Failing task since return code of [/usr/bin/docker run --volume /volumes/bamboo-server/xml-data/build-dir/1736705/WEB-RTA-JOB1:/data 
```

**Comment by gabro** [comment]
It's failing since no tests are present. Add a test :P

**Comment by gabro** [comment]
fixed by #4
